src/gg.regression.unity.bots/Runtime/Scripts/BotManagement/IRGBotEntry.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+
+
+[Serializable]
+public class IRGBotEntry
+{
+    public string botName;
+    public MonoScript monoScript;
```

Comment:
This file prevents the project from building outside the editor.  We should not reference/use anything related to `UnityEditor` in the runtime files.